client/main.lua

```diff
@@ -26,6 +32,35 @@ RegisterNetEvent('qb-printer:client:SpawnPrinter', function()
     SetEntityAsMissionEntity(obj)
 end)
 
+RegisterNetEvent("qb-printer:use")
+AddEventHandler('qb-printer:use', function()
+    SendNUIMessage({
+        action = "start"
+    })
+    SetNuiFocus(true, true)
+end)
+
+-- Threads
+CreateThread(function()
+    while true do
+        local ped = PlayerPedId()
+        local pos = GetEntityCoords(ped)
+        for k, v in pairs(Config.PrinterModels) do
```

Comment:
```suggestion
        for _, v in pairs(Config.PrinterModels) do
```
Use `_` for the unused loop variable to fix linting.